doc/api/modules.md

```diff
@@ -415,7 +415,7 @@ NODE_MODULES_PATHS(START)
 4. while I >= 0,
    a. if PARTS[I] = "node_modules", GOTO d.
    b. DIR = path join(PARTS[0 .. I] + "node_modules")
-   c. DIRS = DIR + DIRS
+   c. DIRS = DIRS + DIR
```

Comment:
If change applies, does it mean that the most nested module will be loaded first?

Reply:
Yes.